DEVELOPER_RESOURCES.md

```diff
@@ -0,0 +1,63 @@
+# KubeArchive Developer Resources
+
+## Scope
+This page provides links to resources useful to a KubeArchive developer.
+
+KubeArchive is a utility that stores Kubernetes resources off of the
+Kubernetes cluster. This enables users to delete those resources from
+the cluster without losing the information contained in those resources.
+
+## Communication
+Chat rooms and email addresses for KubeArchive communication.
+
+TBD
+
+## Dashboards
+TBD
+
+## Development Resources
+
+### Git Repository
+Git repository for the KubeArchive source code.
+
+* [kubearchive/kubearchive](https://github.com/kubearchive/kubearchive)
+
+### Go Programming Language
+Suggested resources for working with the Go programming language.
+
+* [Exercism learning track for Go](https://exercism.org/tracks/go)
+* [Golang By Example - Advanced Tutorial](https://golangbyexample.com/golang-comprehensive-tutorial/)
+* [Ardan Labs Ultimate Go Tour](https://tour.ardanlabs.com/tour/eng/list)
+* [Dave Cheney's Practical Go: Real Worl Advice Guide](https://dave.cheney.net/practical-go/presentations/qcon-china.html)
+
+### Add-ons
+Add-ons the KubeArchive uses.
```

Comment:
I would call these "tools" or "related software" rather than "add-ons".

Reply:
Renamed to "Tools".